frontend/src/app/testQueue/TestCardForm/TestCardForm.tsx

```diff
@@ -116,12 +116,13 @@ const TestCardForm = ({
   const { patientFullName } = useTestOrderPatient(testOrder);
 
   const whichAOEFormOption = useAOEFormOption(state.deviceId, devicesMap);
-
   // AOE responses required for HIV positive result
   const hivAOEResponsesRequired =
     whichAOEFormOption === AOEFormOption.HIV &&
     state.testResults.some(
-      (x) => x.diseaseName === "HIV" && x.testResult === TEST_RESULTS.POSITIVE
+      (x) =>
+        x.diseaseName === MULTIPLEX_DISEASES.HIV &&
```

Comment:
Realized we had an enum that sets the name of the disease as received from GQL, so opting to use that rather than the loose strings